src/context.ts

```diff
@@ -375,6 +375,9 @@ export class Context<
     return this.newResponse(null, status)
   }
 
+  /** @deprecated
+   * Use `streamText()` in `hono/helper/streaming` instead of `c.streamText()`. The `c.streamText()` will be removed in v4.
```

Comment:
You can import it from `hono/streaming`. So, `hono/helper/streaming` should be `hono/streaming`.